### ISSUE TITLE
retroarch@1.8.8: hash check failed

### ISSUE THREAD

**Issue by Eracat1** [lukesampson/scoop-extras#4150]
\> scoop update retroarch
retroarch: 1.8.7 -> 1.8.8
WARN  Scoop uses 'aria2c' for multi-connection downloads.
WARN  Should it cause issues, run 'scoop config aria2-enabled false' to disable it.
Updating 2 outdated apps:
Updating 'retroarch' (1.8.7 -> 1.8.8)
Downloading new version
Starting download with aria2 ...
[...]
Download: (OK):download completed.
Checking hash of RetroArch.7z ... ERROR Hash check failed!
App:         extras/retroarch
URL:         https://buildbot.libretro.com/stable/1.8.8/windows/x86_64/RetroArch.7z
First bytes: 37 7A BC AF 27 1C 00 04
Expected:    bed3948cd744a0a498f1f323e32d2251886e87eabb7431e3b24b7682650d17b2
Actual:      2caaaf2b2a6ecfb84e4a3445ac1fe635f9432dba53c1fefeb26d0619e3f69094

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/lukesampson/scoop-extras/issues/new?title=retroarch%401.8.8%3a+hash+check+failed

**Comment by Eracat1** [comment]
I have tried redownloading the file multiple times, including on my phone on a different network, and I get the exact same hash every time (2caaaf2b2a6ecfb84e4a3445ac1fe635f9432dba53c1fefeb26d0619e3f69094).
I appear to get the same 2 hashes as in #4146, but swapped. Any idea why?

**Comment by joegreene** [comment]
I wrote the previous issue and I'm getting the same issue as well (hashes are swapped). Ran the steps provided to verify this. Problem goes away if the `64bit` property of the app manifest goes back to the following:

```javascript
"64bit": {
    "url": "https://buildbot.libretro.com/stable/1.8.8/windows/x86_64/RetroArch.7z",
    "hash": "2caaaf2b2a6ecfb84e4a3445ac1fe635f9432dba53c1fefeb26d0619e3f69094"
},
```

I'm not sure if something weird happened on my end before to cause that issue earlier (which the bot caught as an OK), but in case case I made another issue to have this become auto-fixed by the bot. I think the bot can only parse the text of the issue properly if you only keep the output and nothing else.
